README.md

```diff
@@ -6,9 +6,9 @@
 | :--:    | :---:               | :---:      | :---:            | :---:                 | :---:                     |
 | v0.10.x | **End-of-Life**     | -          | -                | 2015-10-01            | 2016-10-31                |
 | v0.12.x | **End-of-Life**     | -          | -                | 2016-04-01            | 2016-12-31                |
-| [4.x][] | **Maintenance LTS** | [Argon][]  | 2015-10-01       | 2017-04-01            | April 2018                |
+| [4.x][] | **Maintenance LTS** | [Argon][]  | 2015-10-01       | 2017-04-01            | 2017-04-31                |
```

Comment:
Don't think April 31 exists 😉

Reply:
Also i think it should be 2018. :smile_cat: